docker/generic/start_proxy.py

```diff
@@ -576,6 +576,22 @@ def make_argparser():
         If disabled, JWKS fetching is done when authenticating the JWT, the fetching will add
         to the request processing latency. Default is enabled.'''
     )
+    parser.add_argument(
+        '--jwks_async_fetch_fast_listener',
+        action='store_true',
+        default=False,
+        help='''
+        Only apply when jwks_async_fetch is enabled. This flag determines if the envoy will wait
+        for jwks_async_fetch to complete before binding the listener port. If false, it will wait.
+        Default is false.'''
+    )
+    parser.add_argument(
```

Comment:
Why not enable them by default? 

If not, each JWT needs verification computation?

Reply:
Agreed that it should be safe to enable it by default.  But it is a behavior change, you never know what will happen to an existing product.  Prefer to enable it by users.

Reply:
Well,  I may use a separate pr to enable it.  Since it needs to modify many tests too and in case, it needs to rollback,  just rollback that pr.